design-docs/001-virtual-chunk-improvements.md

```diff
@@ -0,0 +1,163 @@
+# Improvements to virtual chunks handling
+
+## What we need
+
+1. Virtual chunks can be stored in all supported object stores
```

Comment:
I know Rich Signell has used Kerchunk on local filesystems. Do you think virtual chunk storage needs to be limited to object stores?

Reply:
No it doesn't, eventually we can support filesystems

Reply:
Isn't it already supported?

Reply:
Yeah, we have support for filesystem meant mostly for tests. We won't drop it.